vm/src/obj/objrange.rs

```diff
@@ -355,8 +353,10 @@ impl PyValue for PyRangeIterator {
 
 type PyRangeIteratorRef = PyRef<PyRangeIterator>;
 
-impl PyRangeIteratorRef {
-    fn next(self, vm: &VirtualMachine) -> PyResult<BigInt> {
+#[pyimpl]
+impl PyRangeIterator {
+    #[pymethod(name = "__next__")]
```

Comment:
What we usually do for methods that need a `PyRef` is keep it in the `pyimpl` and change the `self` parameter to `zelf: PyRef<Self>`. If you want to do that than feel free, otherwise this is good to merge.

Reply:
I am a little bit confused. It is not about `next` but about `iter`, right? like:

```rust
fn iter(zelf: PyRef<Self>, _vm: &VirtualMachine) -> Self
```

Reply:
Correct, I guess I misclicked where I wanted to place the review. I meant that you can move the `iter` method into the same `impl` block as `next`.

Reply:
Thanks, I fixed it